source/agora/crypto/ECC.d

```diff
@@ -129,17 +129,34 @@ public struct Scalar
 
     ***************************************************************************/
 
+    public void toString (scope void delegate(scope const(char)[]) @safe sink)
+        const
+    {
+        FormatSpec!char spec;
```

Comment:
In this case, what's the default format string?

Reply:
`'s'` is the default